<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>491</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Search database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>279</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Search by</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the time interval to search in the database. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>date:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>from:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="qd_startDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>to:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateEdit" name="qd_endDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QPushButton" name="pb_dateSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>keywords:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="CheckBoxes">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tick the keyword type that the search will be performed. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="3">
              <widget class="QLineEdit" name="LE_animal"/>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="check_marking">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Marking:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3" rowspan="2">
              <widget class="QComboBox" name="cB_nuclide">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Operator:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="check_name">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="check_nuclide">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="4">
              <widget class="QPushButton" name="pb_comboSearch">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Animal ID:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" rowspan="2">
              <widget class="QCheckBox" name="check_Inpath">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="check_op">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_animal">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Radionuclide:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Study name:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" rowspan="2" colspan="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Injection path:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QComboBox" name="cB_op">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="LE_marking"/>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="LE_name"/>
             </item>
             <item row="6" column="3">
              <widget class="QComboBox" name="cB_inj_path">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>278</width>
        <height>413</height>
       </rect>
      </property>
      <attribute name="label">
       <string/>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Studies found</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableView" name="TV_studies">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The table containing the results found. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qd_startDate</tabstop>
  <tabstop>qd_endDate</tabstop>
  <tabstop>pb_dateSearch</tabstop>
  <tabstop>check_name</tabstop>
  <tabstop>LE_name</tabstop>
  <tabstop>check_animal</tabstop>
  <tabstop>LE_animal</tabstop>
  <tabstop>check_marking</tabstop>
  <tabstop>LE_marking</tabstop>
  <tabstop>check_op</tabstop>
  <tabstop>cB_op</tabstop>
  <tabstop>check_nuclide</tabstop>
  <tabstop>cB_nuclide</tabstop>
  <tabstop>check_Inpath</tabstop>
  <tabstop>cB_inj_path</tabstop>
  <tabstop>pb_comboSearch</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>TV_studies</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
